git_buildpackage

```diff
@@ -162,10 +172,32 @@ def create_source_package(repo_dir, output_dir, revision, build_args):
     # Since gbp is the "heart" of this service lets force interactive mode
     safe_run(command, cwd=repo_dir, interactive=True)
 
+    # Lets find the .dsc
+    #if not os.path.exists(
+    found_dsc = False
+    if append_git_id or append_git_url:
+        for fname in os.listdir(absoutput_dir):
+            if  fnmatch.fnmatch(fname, '*.dsc'):
+                input_file = os.path.join(absoutput_dir, fname)
+                found_dsc = True
+                with open(input_file, 'a') as dsc_file:
+                    if append_git_id:
+                        dsc_file.write('Git-Id: ' + git_id)
```

Comment:
Daniel, I've noticed that our existing TAGs give lintian warnings. I'll check out if there is some override.